lib/default.nix

```diff
@@ -5,13 +5,16 @@
  */
 let
 
-  callLibs = file: import file { inherit lib; };
+  fixedPoints = import ./fixed-points.nix {};
```

Comment:
Do `inherit (import ./fixed-points.nix {}) makeExtensible!`.

Reply:
Ah yeah that sounds better